lib/api/endpoints/experiment.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+module API
+  module Endpoints
+    class Experiment < Grape::API
+      desc 'Record an experiment'
+      params do
+        requires :name, type: String, desc: 'Name of experiment'
+        requires :user_id, type: Integer, desc: 'ID of user creating exp'
```

Comment:
+eriment

I hate being nitpick on this stuff but it's worth it for the docs :poodle: